modules/extensions-mule-support/src/main/java/org/mule/runtime/module/extension/mule/internal/loader/parser/BaseMuleSdkExtensionModelParser.java

```diff
@@ -22,7 +22,7 @@
  *
  * @since 4.5.0
  */
-abstract class BaseMuleSdkExtensionModelParser {
+public abstract class BaseMuleSdkExtensionModelParser {
```

Comment:
why? Looks like all implementations are in the same package

Reply:
I have some new ones that are in the sub-package parser.**metadata**, Java doesn't treat it as the same package, hence the visibility issue.